package.json

```diff
@@ -90,7 +90,102 @@
     "pull-ws": "^3.3.2",
     "json-schema": "^0.4.0",
     "simple-get": "^4.0.1",
-    "@storybook/**/ast-types": "^0.14.2"
+    "@storybook/**/ast-types": "^0.14.2",
+    "typescript@~4.4.0": "patch:typescript@npm:4.4.4#.yarn/patches/typescript-npm-4.4.4-3fedcc07a3.patch",
```

Comment:
Hmm... so the docs on the `patch:` protocol seem to indicate that this not only works for `resolutions` but also for `*dependencies` fields. Since `typescript` is a dependency that we list here should this be moved to the `dependencies` section? Or did the `yarn patch-commit` automatically add this line to `resolutions`?

Reply:
patch-commit automatically adds it to the resolutions

Reply:
I can attempt to move these to dependencies if it makes more sense and works. i'll give it a shot

Reply:
@mcmire it does look like this will work, but i'm wondering if it still makes sense to use resolutions instead of the dependencies fields. For one the version of the dependency that is actually installed is "hidden" or at least placed inside the patch string. The other reason is that it keeps all patches together in the resolutions field. Also patch-commit adds these resolutions to the package.json in the resolutions section -- which will require more overhead from the engineer writing patches.

Reply:
Okay, no problem. That's fine.